.github/workflows/build.yml

```diff
@@ -12,10 +12,18 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: [ '7.1', '8.0' ]
```

Comment:
we have a fast build, what about adding even 7.2, 7.3 and 7.4?

Reply:
I agree with you!